debugProtocol.json

```diff
@@ -2752,6 +2903,16 @@
 				"endColumn": {
 					"type": "integer",
 					"description": "Optional end column of the range covered by this scope."
+				},
+				"kind": {
+					"type": "string",
+					"description": "Optional hint describing the contents of this scope.",
+					"_enum": [ "arguments", "locals", "registers" ],
```

Comment:
Could we add an "other" as a fallback. I know it is optional but...

Reply:
we never did this in DAP and we don't want to start this now...

Reply:
Nope. Not if it is not normal practice

Reply:
So your "nope" means you are agreeing or disagreeing?

Reply:
@weinand I am sorta agreeing with you. The reason I even had the original comment is if it is not "arguments" or "locals" or "registers", then what is it. You have to omit the enum to say it is something else. You know the rules and norms better than I do.

Reply:
Since the DAP protocols is always backward compatible, we cannot rely on the use of "other"  because old implementations of DAs don't use the "kind" attribute anyway. So we have to specify what it means if the "kind" is missing. This will cover the "other" cases.
Having both a missing "kind" and a kind value "other" makes things more complex without adding much value.